src/models/Destination.js

```diff
@@ -0,0 +1,51 @@
+export default (sequelize, DataTypes) => {
+  const Destination = sequelize.define('Destination', {
```

Comment:
Also is the creation of a new schema called *Destination*  necessary? all these can fit into the *Request* model. Then again, *Destination* isn't descriptive enough of what the schema is about if at all it is necessary. From what I can see, the only difference is the addition of a *multiDestination* row, which can easily fit into the Request table. *multiFlightDate* too can still fit into the Request table. My point is, the multi-city request is a request and should be in the request table. Great job!!!!

Reply:
that make sense, will try and implement that

Reply:
done , thanks